app/helpers/confirm_details_helper.rb

```diff
@@ -3,6 +3,7 @@
 module ConfirmDetailsHelper
   def trainee_section_update_path(section_key, trainee)
     routes = {
+      "diversity" => "trainee_diversity_confirm_path",
```

Comment:
Not sure you meant to add this?

Reply:
@AbigailMcP Due to the changes in the routes file, this kept throwing error, seemed to be missing from the list.